utils/build_swift/tests/expected_options.py

```diff
@@ -175,7 +175,8 @@
     'validation_test': None,
     'verbose_build': False,
     'watchos': False,
-    'watchos_all': False
+    'watchos_all': False,
+    'test_paths': []
```

Comment:
If you wouldn't mind I'd prefer to keep these defaults in alphabetical order.

Reply:
Sure, I didn't realize that this was the case for some reason...